include/groonga/util.h

```diff
@@ -137,6 +137,39 @@ grn_inspect_indented(grn_ctx *ctx,
  */
 GRN_API grn_obj *
 grn_inspect_limited(grn_ctx *ctx, grn_obj *buffer, grn_obj *obj);
+/**
+ * \brief Inspect specified object and return the name of the specified object.
+ *
+ * This function inspects the given \p obj and writes its name into \p buffer.
+ * If \p obj has a name, that name is copied into \p buffer.
+ * Otherwise:
+ * - If \p obj's id is a \ref GRN_ID_NIL, the `(nil)` is stored in \p buffer.
+ * - If \p obj exists but has no name, the `(anonymous:xx)` is stored.
```

Comment:
```suggestion
 * - If \p obj exists but has no name, the `(anonymous:ID)` is stored.
```

I think `ID` is better than `xx`.
It is easier to understand that an ID is set.

Reply:
[2e091f7](https://github.com/groonga/groonga/pull/2354/commits/2e091f7bc8ba4d538c784d1fcde8c54a5009d7d2) Sure!